seadssite/views.py

```diff
@@ -41,7 +41,7 @@ def register(request):
 
     # if a user "POST"s, check phone number, provider and user/profile form(models)
     if request.method == 'POST':
-        print ("user is about to register")
+        print("user is about to register")
```

Comment:
Agreed, although I agree with @iangudger  about logging, I  just dont like random print statements littering the code base.